state_management/java/http/order-processor/src/main/java/com/service/OrderProcessingServiceApplication.java

```diff
@@ -0,0 +1,84 @@
+package com.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+public class OrderProcessingServiceApplication {
+    private static final Logger LOGGER = LoggerFactory.getLogger(OrderProcessingServiceApplication.class);
+
+    private static final String DAPR_STATE_STORE = "statestore";
+    private static String DAPR_HOST = System.getenv().getOrDefault("DAPR_HOST", "http://localhost");
+    private static String DAPR_HTTP_PORT = System.getenv().getOrDefault("DAPR_HTTP_PORT", "3500");
+    private static final HttpClient httpClient = HttpClient.newBuilder()
+            .version(HttpClient.Version.HTTP_2)
+            .connectTimeout(Duration.ofSeconds(10))
+            .build();
+
+    public static void main(String[] args) throws IOException, URISyntaxException, InterruptedException {
+        URI baseUrl = new URI(DAPR_HOST+":"+DAPR_HTTP_PORT);
+        URI stateStoreUrl = new URI(baseUrl + "/v1.0/state/"+DAPR_STATE_STORE);
+        while (true) {
+            Random random = new Random();

```

Comment:
Please see @amulyavarote's latest PR on changing random to well known for loop.  She decided random is going to block testability, and I agree.

As a part of change please check you've removed unused imports for java.util.Random